text/87_interacting_with_stakeholders.md

```diff
@@ -10,7 +10,7 @@ As a software tester, you will have specific challenges when interacting with ot
 
 A __stakeholder__ is any person who has a direct interest in the product.  The specific kind of interest may vary based on the person and their role.  For example, the customer of a product - that is, the person who pays for it - will have a vested interest in paying as little as possible for the product.  Upper management and marketers have the exact opposite goal - they want customers to pay as much for it as possible, within epsilon of the price where people would choose a competitor.  Developers may have an interest in seeing the product written correctly, or using a language or framework that they prefer.  Quality assurance personnel may care about the quality of a product; assessors may care that the product meets all of its legal requirements; the list goes on and on.
 
-Note that stakeholders have a _direct_ interest in the product.  In the philosophical sense, many others will have an interest in the product, or at least will be impacted by it.  For example, the owners of the coffee shop next door will hope that developers start working late and thus buying more of that sweet, sweet caffeine that comes with their coffee, thus improving their bottom line.  The spouses of the testers may want there to be fewer defects so that the testers can spend more time with their family.  However, these are not direct interests in the product, and it could be argued that any human being is somehow impacted by the project (the Prime Minister of Luxembourg may hope to have your nation's GDP go up in order to improve their position in trade negotiations, and a successful launch of your product will help that goal!)  Although it can be a fuzzy boundary, in general if a person is not directly familiar with your product, they are in all likelihood not a stakeholder.
```

Comment:
"the owners ... will hope that developers start working late and thus buying more..." 

"thus buying" sounded funny